fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/SplitHelperTest.java

```diff
@@ -393,13 +441,13 @@ public void saveWithSplitAndIncompleteVersions(SplitHelperTestConfig testConfig)
 
             commit(context);
             versionstamp = context.getVersionStamp();
-            if (!testConfig.omitUnsplitSuffix) {
+            if (!testConfig.omitUnsplitSuffix && !testConfig.isDryRun) {
```

Comment:
Why is this necessary? It seems like the whether the config is dry-run shouldn't affect this

Reply:
I think `versionStamp` is set when `transaction.commit()`, so if it is dry run, no versionStamp is set.

Reply:
Ah, i see. I’m actually a little surprised this returns `null` now if `omitVersionSuffix` is set, but maybe I’m misremembering how that value in the context gets populated. I would have thought it would show up on every commit that isn’t read only. Hmm